packages/composite-checkout/src/components/billing-fields.js

```diff
@@ -0,0 +1,221 @@
+						<Field
+							id="billing-city"
+							type="Text"
+							label={ localize( 'City' ) }
+							error={ billingCityError }
+							errorMessage={ localize( 'This is a required field' ) }
+							value={ billingCity }
+							onChange={ cityOnChange }
+						/>
+
+						<Field
+							id="billing-state"
+							type="Text"
+							label={ localize( 'State' ) }
+							error={ billingStateError }
+							errorMessage={ localize( 'This is a required field' ) }
+							value={ billingState }
+							onChange={ stateOnChange }
+						/>
+					</FieldRow>
+				</React.Fragment>
+			) }
+
+			<FieldRow gap="4%" columnWidths="48% 48%">
+				<Field
+					id="billing-zip-code"
+					type="Text"
+					label={ localize( 'Zip code' ) }
+					error={ billingZipCodeError }
+					errorMessage={ localize( 'This is a required field' ) }
+					value={ billingZipCode }
+					onChange={ zipCodeOnChange }
+				/>
+
+				<Field
+					id="billing-country"
+					type="Text"
+					label={ localize( 'Country' ) }
+					error={ billingCountryError }
+					errorMessage={ localize( 'This is a required field' ) }
+					value={ billingCountry }
+					onChange={ countryOnChange }
+				/>
+			</FieldRow>
+
+			<FormField
+				id="billing-phone-number"
+				type="Number"
+				label={ localize( 'Phone number (Optional)' ) }
+				error={ billingPhoneNumberError }
+				errorMessage={ localize( 'This is a required field' ) }
+				value={ billingPhoneNumber }
+				onChange={ phoneNumberOnChange }
+			/>
+
+			{ isElligibleForVat() && (
+				<FormField
+					id="billing-vat-id-number"
+					type="Text"
+					label={ localize( 'VAT identification number' ) }
+					description={ localize( 'For businesses only' ) }
+					error={ billingVatIdError }
+					errorMessage={ localize( 'This is a required field' ) }
+					value={ billingVatIdNumber }
+					onChange={ vatIdOnChange }
+				/>
+			) }
+		</BillingFormFields>
+	);
+}
+
+const BillingFormFields = styled.div`
+	margin-bottom: 16px;
+`;
+
+const FormField = styled( Field )`
+	margin-top: 16px;
+	:first-child {
+		margin-top: 0;
+	}
+`;
+
+const FieldRow = styled( GridRow )`
+	margin-top: 16px;
+
+	first-child {
+		margin-top: 0;
+	}
+`;
+
+function isDomainRelatedField( items ) {
+	if ( items.find( item => item.type === 'domain' ) ) {
+		return true;
+	}
+
+	return false;
+}
+
+function isElligibleForVat() {
+	return false;
+}
+
+function BillingFormSummary( { paymentData } ) {
+	const localize = useLocalize();
+	const {
+		billingName = '',
+		billingAddress = '',
+		billingCity = '',
+		billingState = '',
+		billingZipCode = '',
+		billingCountry = '',
+		billingPhoneNumber = '',
+		billingVatIdNumber = '',
+	} = paymentData || {};
+
+	return (
+		<React.Fragment>
+			{ billingName } <br />
+			{ billingAddress } <br />
+			{ billingCity }, { billingState } <br />
+			{ billingZipCode }, { billingCountry } <br />
+			<br />
+			{ billingPhoneNumber }
+			{ isElligibleForVat() && (
+				<React.Fragment>
+					<br />
```

Comment:
Would it be more semantic if we wrap these fields in `<p>` tags instead of using `<br>`? Then we could style them with CSS (we could also give some `p` tags a bigger margin instead of having two `<br>` tags on line 210).

Reply:
Good idea. I added paragraphs as wrappers around the various blocks of copy.